src/rawtoaces_core/rawtoaces_core_priv.h

```diff
@@ -11,25 +11,25 @@ namespace rta
 namespace core
 {
 
-std::vector<double> cctToxy( const double &cctd );
+std::vector<double> CCT_to_xy( const double &cctd );
 
-void scaleLSC( const SpectralData &camera, SpectralData &illuminant );
+void scale_LSC( const SpectralData &camera, SpectralData &illuminant );
 
 std::vector<double>
-calCM( const SpectralData &camera, const SpectralData &illuminant );
+calculate_CM( const SpectralData &camera, const SpectralData &illuminant );
 
-std::vector<Spectrum>
-calTI( const SpectralData &illuminant, const SpectralData &training_data );
+std::vector<Spectrum> calculate_TI(
+    const SpectralData &illuminant, const SpectralData &training_data );
 
 std::vector<double>
-calWB( const SpectralData &camera, SpectralData &illuminant );
+_calculate_WB( const SpectralData &camera, SpectralData &illuminant );
```

Comment:
calling this one `_calculate_WB` to make it compile. There is another calculate_WB without arguments. Let me know what is the better name for one or another so they are deconflicted.